class AssistError(Exception):
    pass

class AssistChargeError(AssertionError):
    pass
